package/MDAnalysis/analysis/rdf.py

```diff
@@ -156,3 +157,152 @@ def _conclude(self):
         rdf = self.count / (density * vol * self.n_frames)
 
         self.rdf = rdf
+
+
+
+"""
+Site-specific Radial Distribution Functions --- :mod:`MDAnalysis.analysis.rdf_s`
```

Comment:
Add this text to the text at the top. I will clean up the formatting --- that's easier to show than to explain.

Fix rdf_s --> rdf

(You can have a look at the text in some of the other modules.)